src/status_im/subs/wallet/activities_test.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.subs.wallet.activities-test
+  (:require
+    [cljs.test :refer [is testing]]
+    [re-frame.db :as rf-db]
+    status-im.subs.root
+    status-im.subs.wallet.collectibles
+    [test-helpers.unit :as h]
+    [utils.re-frame :as rf]))
+
+(h/deftest-sub :wallet/all-activities
+  [sub-name]
+  (testing
+    "Return the activities list from wallet data"
+    (swap! rf-db/app-db assoc-in
+      [:wallet :activities]
+      [{:id 1 :name "Transaction1"}
+       {:id 2 :name "Transaction2"}])
+    (let [result (rf/sub [sub-name])]
+      (is (= [{:id 1 :name "Transaction1"} {:id 2 :name "Transaction2"}] @result)))))
+
+(h/deftest-sub :wallet/activities-for-current-viewing-account
+  [sub-name]
+  (testing
+    "Return activities filtered and grouped by account and dates"
+    (swap! rf-db/app-db assoc
+      {:wallet/all-activities                  [{:sender "acc1" :recipient "acc2" :timestamp 1588291200}
```

Comment:
I think this `assoc` is incorrect because the app-db paths are `[:wallet :activities]` and `[:wallet :current-viewing-account-address]`: